homeassistant/components/axion_dmx/config_flow.py

```diff
@@ -0,0 +1,73 @@
+"""Config flow for Axion Lighting integration."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from libaxion_dmx import AxionDmxApi
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import _LOGGER, CONF_CHANNEL, CONF_LIGHT_TYPE, DOMAIN
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_CHANNEL): vol.All(vol.Coerce(int), vol.Range(min=1)),
+        vol.Required(CONF_LIGHT_TYPE): vol.In(
+            ["White", "Tunable White", "RGB", "RGBW", "RGBWW"]
+        ),
```

Comment:
Our device is based on DMX512 protocol, and the communication on the line is only one way based, so we cannot detect this, since we don't have the return response from the light.

Reply:
Please make it a selector with a translation key for this